### DISCUSSION TITLE
Remapping keys to a command that requires input

### DISCUSSION THREAD

**Discussion by MoonKraken** [helix-editor/helix#3837] [Q&A]
Hello - I'm trying to do this in `config.toml` to find the first occurence of `r` on the current line:

```
[keys.normal]
'A-"' = [
  "goto_line_start",
  "find_next_char",
  "r"
]
```

But it doesn't seem to be working. Generally speaking for commands that require a character after them, ie `find_next_char`, how do I specify that character in key remappings?

**Comment by MoonKraken** [comment]
After poking around it seems like this needs to be implemented for this to work, let me know if that's wrong: https://github.com/helix-editor/helix/issues/1383

**Reply by Omnikar** [reply]
Yes, that is correct.
